runtime/ddr/CMakeLists.txt

```diff
@@ -44,6 +44,13 @@ if(OMR_TOOLCONFIG STREQUAL "gnu")
 	target_compile_options(j9ddr_misc PRIVATE  -femit-class-debug-always)
 endif()
 
+
+if(OMR_OS_AIX AND (OMR_TOOLCONFIG STREQUAL "xlc"))
```

Comment:
Does this need the `xlc` check?  UMA had enabled unconditionally on aix

Reply:
Its not entirely clear, since we only build with xlc on aix. Will have to dig into docs to see if gcc will accept this

Reply:
Any update on this?

Reply:
Looks like gcc will accept this. Removing xlc check